lib/features/children/add_member/repository/add_member_repository.dart

```diff
@@ -20,13 +20,27 @@ class AddMemberRepositoryImpl with AddMemberRepository {
   @override
   Future<void> addMembers(List<Member> members, {required bool isRGA}) async {
     final profilesJsonList = members.map((member) => member.toJson()).toList();
+    final hasChild = members.any((member) => member.isChild);
     final body = {
       'profiles': profilesJsonList,
       'allowanceType': isRGA ? 0 : 1,
     };
 
     await apiService.addMember(body);
     _memberAddedStreamController.add(null);
```

Comment:
I wouldn't even do the hasChild check. Would just do the event after 1 second. Delete the other event emit.